app/views/supplementary/summary/additional_documentation_section.scala.html

```diff
@@ -14,34 +14,33 @@
  * limitations under the License.
  *@
 
-@import forms.supplementary.DocumentsProduced
+@import models.declaration.supplementary.DocumentsProducedData
 @import models.viewmodels.HtmlTableRow
-
-@(additionalDocumentation: Option[DocumentsProduced])(implicit messages: Messages)
+@(additionalDocumentationData: Option[DocumentsProducedData])(implicit messages: Messages)
 
 @components.summary_list(Some(messages("supplementary.summary.additionalDocumentation.header"))) {
     @components.table_row_no_change_link(HtmlTableRow(
         label = messages("supplementary.summary.additionalDocumentation.documentTypeCode"),
-        value = additionalDocumentation.flatMap(_.documentTypeCode)
+        value = additionalDocumentationData.flatMap(_.documents.map(_.documentTypeCode))
     ))
     @components.table_row_no_change_link(HtmlTableRow(
         label = messages("supplementary.summary.additionalDocumentation.documentId"),
-        value = additionalDocumentation.flatMap(_.documentIdentifier)
+        value = additionalDocumentationData.map(_.documents.map(_.documentIdentifier))
```

Comment:
This comment about `flatMap` is for whole this file. Here you use a `map` and the same in lines: 32, 36, 40, 44